src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -902,21 +905,25 @@ protected RequestLineStatus TakeStartLine(SocketInput input)
                 Method = method;
                 RequestUri = requestUrlPath;
                 QueryString = queryString;
+                RawTarget = rawTarget;
                 HttpVersion = httpVersion;
 
                 bool caseMatches;
-
-                if (!string.IsNullOrEmpty(_pathBase) &&
-                    (requestUrlPath.Length == _pathBase.Length || (requestUrlPath.Length > _pathBase.Length && requestUrlPath[_pathBase.Length] == '/')) &&
-                    RequestUrlStartsWithPathBase(requestUrlPath, out caseMatches))
+                if (RequestUrlStartsWithPathBase(requestUrlPath, out caseMatches))
```

Comment:
Are these changes related to RawTarget, or just bugs we found when testing RawTarget?